autocomplete.js

```diff
@@ -192,12 +194,15 @@ function Autocomplete(_settings) {
         event.preventDefault();
         if (_currentIndex > -1) {
           _currentListEls[_currentIndex].classList.remove("is-selected");
+          _currentListEls[_currentIndex].setAttribute("aria-selected", "false");
         }
         _currentIndex = Math.max(
           _currentIndex === -1 ? _currentListEls.length - 1 : _currentIndex - 1,
           0
         );
         _currentListEls[_currentIndex].classList.add("is-selected");
+        _currentListEls[_currentIndex].setAttribute("aria-selected", "true");
+        input.setAttribute("aria-owns", _currentListEls[_currentIndex].getAttribute("id"));
```

Comment:
The `aria-selected` change is good, but looking at the example again, the `aria-owns` on input should just always be the list of results. The individual result id should be updated to the input's `aria-activedescendant`.